server/src/game/event/on_grave_added.rs

```diff
@@ -0,0 +1,17 @@
+use crate::game::{grave::Grave, Game};
+
+
+pub struct OnGraveAdded{
+    pub grave: Grave
+}
+impl OnGraveAdded{
+    pub fn new(grave: Grave) -> Self{
+        Self{ grave }
+    }
+    pub fn invoke(self, game: &mut Game){
+        game.on_grave_added(self.grave);
+    }
+    pub fn create_and_invoke(game: &mut Game, grave: Grave){
```

Comment:
Why have a specific "do two things" function when you can just do `OnGraveAdded::new().invoke(grave)`. Seems a little strange.
This is also inconsistent because OnAnyDeath doesn't have this

Reply:
Why not.
Ill put it on OnAnyDeath as well.
Its easier to call

Reply:
If I put it on OnAnyDeath it wont be used.

Reply:
create_and_invoke is literally longer to write. But idk ig if it's more convenient for you that's fine. At the end of the day you're the one using these the most

Reply:
done your way. you are right